### ISSUE TITLE
No Textures

### ISSUE THREAD

**Issue by Arka326** [TheRexYo/HiveCiv-Mod#2]
Textures for units and techs generate as needed.

**Comment by TheRexYo** [comment]
Yeah, I'm not the greatest artist but I'll get there. Probably once all the content is done, because gameplay is kinda limited right now.
